lib/utils.js

```diff
@@ -72,8 +71,18 @@ function addRoute (app, path) {
   app.get(path, Routes.facilityCodeLookup)
 }
 
+// express 4.14.1 specific
 function removeRoute (app, path) {
-  removeRouteDep(app, path)
+  let counter = 0
+  const stack = app._router.stack
+  stack.forEach((layer) => {
+    // console.log(path)
+    // console.log(layer)
+    if(layer && path.match(layer.regexp) && layer.name === 'bound dispatch') {
```

Comment:
I think that using `layer.route.path` to match the extact path might be better. There could potentially be multiple routes whose regex matches the supplied path, but we only want to remove the specific one that was added by the config.